Rmarkdown/Multiple_Ascending_Dose_PD_binary.Rmd

```diff
@@ -1,303 +1,271 @@
+
+There are two broad categories of PK/PD exploratory plots covered on this page
+
+1. *Exposure and Response vs Time*, stratified by dose. You may also have heard these referred to as longitudinal (meaning over time).
+2. *Response vs Exposure* at a particular time.  For binomial response vs exposure plots, fitting a logistic regression is often helpful, as you will see below.
+
+These plots are displayed below.
+
+## Provide an overview of the data
+
+We start with *Response vs Time*, or longitudinal plots 
+
+Binary data is often summarized by displaying the percent of subjects in the "yes" category, e.g. the percent of "responders," along with 95% confidence intervals.  For generating confidence intervals, it is useful to bin data points by nominal time; confidence intervals should be calculated using the `binom::binom.exact()` function.
+
+### Responder rate (95% CI) by nominal time, colored or faceted by dose
+
+Questions to ask when looking at these plots
+
+* Does the response rate increase/decrease over time? 
+* Does response change with increasing dose?
+* Do you notice a plateau in time?
+* Do you notice a plateau with dose (Emax)?
+
+**Warning:** Even if you don't see a plateau in response, that doesn't mean there isn't one. Be very careful about using linear models for Dose-Response relationships. Extrapolation outside of the observed dose range could indicate a higher dose is always better (even if it isn't).
+
+When overlaying individual binomial data, it is often helpful to stagger the dots and use transparency, so that it is easier to see individual data points.  If you do so, this should be clearly stated.  
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 4, warning = FALSE, message= FALSE} 
+
+  gg <- ggplot(data = pd_data, aes(x = NOMTIME, y = LIDV, color = TRTACT_high2low))
+  gg <- gg + xgx_stat_ci(distribution = "binomial")
+  gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                    units_plot    = time_units_plot)
+  gg <- gg + scale_y_continuous(labels=scales::percent)
+  gg <- gg + labs(y = pd_label, color = trtact_label)  
+  gg <- gg + xgx_annotate_status(status)
+  #gg <- gg + xgx_annotate_filenames(dirs)
+  #if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+  #xgx_save(width,height,dirs,"filename_main",status)
+  print(gg)
+
+```
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE}
+
+  gg <- ggplot(data = pd_data, aes(x=PROFDAY,y=LIDV))
+  gg <- gg + geom_point(aes(x=DAY_jitter,y=LIDV_jitter), size=2, alpha = 0.3)
+  gg <- gg + xgx_stat_ci(conf_level = 0.95, distribution = "binomial", geom = list("line", "ribbon"))
+
+  gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 

```

Comment:
units_dataset = time_units_plot

Because this figure is plotting x = PROFDAY which has units of days

Reply:
changed to time_units_plot now and the figure works without scale_x_continuous